bundler/benchmarks/driver.rb

```diff
@@ -70,6 +72,10 @@ def run_single(path)
     puts "Posting results to Web UI...."
   end
 
+  def generate_digest(path)
+    Digest::SHA2.hexdigest("#{File.read(path)}#{`ruby -v`}")
```

Comment:
@segiddins @indirect Do let me know if there are more dependencies that we should include in the digest. Many thanks! :smile: